### ISSUE TITLE
Creating a basic guide/FAQ fo developers starting out

### ISSUE THREAD

**Issue by SayakMukhopadhyay** [EDCD/EDCD.github.io#8]
Based on a discussion in Discord (https://discord.com/channels/164411426939600896/164411426939600896/884741684850753546), it would be great to have a guide for new developers, on a similar vein as the ELI5 guide.

I am willing to take this up so it would be great if someone can assign me this issue to help me track it.

**Comment by robbyxp1** [comment]
Now out of date, closing
